.circleci/config.yml

```diff
@@ -443,6 +443,18 @@ jobs:
             - run:
                 name: ktlint
                 command: ./gradlew ktlint
+            - run:
+                name: check release runtime Mapbox dependencies to fail fast
+                command: |
+                  curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg
+                  echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+                  sudo apt update && sudo apt install gh -y
```

Comment:
seems this step to install github client is used in other tasks as well, can we share this step? or we can add it to the docker image.

Reply:
I checked the hierarchy of docker images and if I understood it correctly `gh` command should be available already

Reply:
I also remove it from `sdk-e2e-test`